Utilities/ITKv5Preparation/prefer-type-alias-over-typedef.sh

```diff
@@ -0,0 +1,35 @@
+cat > ./typdef_vim_script.vim << EOF
```

Comment:
I used a relative path instead of `/tmp/typdef_vim_script.vim`.

Feel free to change the temporary filename created.